models/mask_generator.py

```diff
@@ -305,7 +305,7 @@ def save_test_images(self, test_display_data, curr_iter):
             save_images.append(save_real_mask_seg)
             save_images.append(save_real_mask)
         write_images(
-            save_images, curr_iter, comet_exp=self.comet_exp, store_im=self.store_image
+            save_images, curr_iter, comet_exp=self.comet_exp, store_im=self.store_image, is_test = is_test
```

Comment:
Line is too long, I don't think `black` would let you do that ;)

Reply:
Oh no you caught me! I edited in jupyterlab :)